app/lib/main.dart

```diff
@@ -74,8 +78,14 @@ Future<void> _startAppInner(Widget app, bool withSentry) async {
     container: mainProviderContainer,
     child: app,
   );
+  if (withAnalytics) {
+    await MatomoTracker.instance.initialize(
+      siteId: Env.matomoSiteId,
+      url: Env.matomoUrl,
+    );
 
-  if (withSentry) {
+    MatomoTracker.instance.setOptOut(optOut: true);
```

Comment:
:+1: 

only problem is now: when we do this, we haven't actually figured out if the user logged in. So we still need a listener on `Acter` (below) that listens to the pref-provider and updates it when the client switches (or logs in) etc.

Then all the others have to do it just change the provider switch and not care of `MatomoTracker.instance.setOptOut` at all, as it would just "happen" by the outer listener.